src/engine/RuntimeInformation.h

```diff
@@ -91,10 +93,10 @@ class RuntimeInformation {
 
   /// Get the time spent computing the operation. This is the total time minus
   /// the time spent computing the children.
-  [[nodiscard]] double getOperationTime() const;
+  [[nodiscard]] std::chrono::milliseconds getOperationTime() const;
```

Comment:
This is the function you changed, so you changed the wrong docstring...

Reply:
Oh right, good catch